archstrike-testing/teamsql/PKGBUILD

```diff
@@ -0,0 +1,25 @@
+# Maintainer: ArchStrike <team@archstrike.org>
+
+pkgname=teamsql
+pkgver=2.6.159
+pkgrel=2
```

Comment:
pkgrel should start at 1

Reply:
Can this be exception since I am also maintaining this on AUR so it is easier to maintain it like this to me?

Reply:
ok, fixed